### PR TITLE
Increase compression level for ResultSet

### PR DESCRIPTION
**Pull request by ConnectedSystems**
Increasing compression level for slightly more manageable ResultSet sizes.

### CODE REVIEW

File: src/io/ResultSet.jl [hunk 1]
```diff
@@ -212,6 +212,7 @@ function combine_results(result_sets...)::ResultSet
                 nrow(all_inputs),
                 size(rs1.seed_log, :timesteps),
                 size(rs1.seed_log, :sites),
+                36 # TODO: Find a way to remove this constant
```
Comment (ConnectedSystems) [inline src/io/ResultSet.jl:215]: Oops, this may have been a defunct change that crept in.

Comment (Zapiano) [reply src/io/ResultSet.jl:215]: I only see one method for this function `setup_logs` and it sccepts 5 arguments: `setup_logs(z_store, unique_sites, n_scens, tf, n_sites)`. I think this line should be removed, no?

Comment (ConnectedSystems) [reply src/io/ResultSet.jl:215]: Yeah I'm trying to work out how it crept in...

**Comment by ConnectedSystems** [comment]
This really was meant to be a one line change, not sure how the older line crept in!

All fixed now.

**Review by Zapiano — APPROVED** [review]
🚀

**Pull request merged**